pages/docs/2_modify/advanced/extending_local_version_comunica.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Using Local Changes to Base Comunica in Comunica Extensions'
+description: 'Guide on how to extend a local version of Comunica or use a local version of Comunica with an existing extension'
+---
+
+Various  [extensions](https://comunica.dev/docs/modify/extensions/) of Comunica exist. 
+These extensions utilize packages from the [base Comunica](https://github.com/comunica/comunica) framework and add additional packages or engine configurations. 
+When working on local changes to the base Comunica and needing to use these changes in an extension, you must link your local development version of Comunica with your extension project.
+This page introduces some methods to achieve this, though other methods are possible.
+
+## Lerna-linker
+
+Lerna-linker is a script designed to facilitate package linking in a Lerna monorepo. It iterates over all packages, executing `yarn unlink` and `yarn link` on each. It then saves all linked packages and runs `yarn link <package>` for each linked package in the Comunica extension.
+
+### Installation
+
+Install the script globally using the following:
+
+```bash
+$ npm install -g lerna-linker
```

Comment:
Could you also add some simple docs to the readme of this package?
And also a link from npm to your github repo.